lib/executor/helpers.go

```diff
@@ -56,19 +56,20 @@ func validateStages(stages []Stage) []error {
 	var errors []error
 	if len(stages) == 0 { //nolint:nestif
```

Comment:
The linter is complaining that this `nolint` directive is no longer needed, so you should remove it.

Reply:
@na-- thanks! done.